shpc/main/modules/base.py

```diff
@@ -348,7 +348,13 @@ def check(self, module_name):
         return self.container.check(module_name, config)
 
     def install(
-        self, name, tag=None, view=None, disable_view=False, force=False, **kwargs
+        self,
+        name,
+        tag=None,
+        extra_view=None,
```

Comment:
If we really wanted to make this an extra thing, we could remove mentions of view from the named arguments and add to kwargs instead, e.g., `extra_view = kwargs.get("extra_view")`. And I'd be OK with still just using `view` for the argument name, but having it populate into `extra_view` so it's more clear for the developer, e.g.,:

```python
extra_view = kwargs.get("view")
```
I kind of like that, because the "default view" comes from settings (unknown possibly to the function caller, who might just want to define a `view`) but for the developer reader, it is nicer to read the variable as `extra_view`. Two birds, one stone!